docs/how-to-guides/installing-uninstalling-extensions.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Installing and Uninstalling Rancher Desktop Extensions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsConstants from '@site/core/TabsConstants';
+
+The **Extensions** feature introduced in the `v1.9.0-tech-preview` release enables you to use **Docker Desktop Extensions** within Rancher Desktop. The feature helps you extend Rancher Desktop's functionality to meet your additional needs. This guide will demonstrate how to install and uninstall Docker extensions in Rancher Desktop.
+
+### Prerequisites
+
+Rancher Desktop Extensions are available as part of release `v1.9.0-tech-preview` or later.
+
+### Installing Extensions
+
+There are two ways in which you can install extensions, a method using the UI and another using the `rdctl extension install` command.
+
+<Tabs groupId="os" defaultValue={TabsConstants.defaultOs}>
+<TabItem value="Windows">
+
+#### Using the UI
+
+1. Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use.
+
+![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/Windows_Extensions.png)
+
+2. When you find an extension you want to use, click on the extension to open the **Detail** view, then click on the **Install** button on the right hand side as seen in the image below. 
+
+![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/Windows_Extensions-Details.png)
+
+3. Please shutdown and restart your Rancher Desktop application for changes to take effect.
```

Comment:
We don't have to shutdown and restart Rancher Desktop to see the changes after installing an extension.